py3-pgcli.yaml

```diff
@@ -36,13 +37,85 @@ pipeline:
       tag: v${{package.version}}
       expected-commit: 46fe654cd16a3af837639dc2c8be60d1b7a038c7
 
-  - name: Python Build
-    runs: python setup.py build
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: python${{range.key}} version of ${{vars.pypi-package}}
+    dependencies:
+      provider-priority: ${{range.value}}
+      runtime:
+        - py${{range.key}}-configobj
+        - py${{range.key}}-typing-extensions
+        - py${{range.key}}-click
+        - py${{range.key}}-sqlparse
+        - py${{range.key}}-cli-helpers
+        - py${{range.key}}-prompt-toolkit
+        - py${{range.key}}-pygments
+        - py${{range.key}}-pgspecial
+        - py${{range.key}}-pendulum
+        - py${{range.key}}-psycopg
+        - py${{range.key}}-setproctitle
+    pipeline:
+      - name: Python Build
+        runs: python${{range.key}} setup.py build
+      - name: Python Install
+        runs: python${{range.key}} setup.py install --prefix=/usr --root="${{targets.contextdir}}"
+      - name: "move usr/bin executables for -bin"
+        runs: |
+          mkdir -p ./cleanup/${{range.key}}/
+          mv ${{targets.contextdir}}/usr/bin ./cleanup/${{range.key}}/
+      - uses: strip
 
-  - name: Python Install
-    runs: python setup.py install --prefix=/usr --root="${{targets.destdir}}"
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}-bin
+    description: Executable binaries for ${{vars.pypi-package}} installed for python${{range.key}}
+    dependencies:
+      runtime:
+        - py${{range.key}}-${{vars.pypi-package}}
+      provides:
+        - py3-${{vars.pypi-package}}-bin
+        - py3-${{vars.pypi-package}}
+        - ${{package.name}}
+      provider-priority: ${{range.value}}
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/usr/
+          mv ./cleanup/${{range.key}}/bin ${{targets.contextdir}}/usr/
+      # Change the interpreter to the appropriate python version
+      - runs: |
```

Comment:
you should not have to do this... almost certain you will not have to if you use the py/pip-build-install pipeline.